src/helperFunctions/install.py

```diff
@@ -160,7 +227,12 @@ def _checkout_github_project(github_path, folder_name):
         raise InstallationError('Repository creation failed on folder {}\n {}'.format(folder_name, clone_url))
 
 
-def load_main_config():
+def load_main_config() -> configparser.ConfigParser:
```

Comment:
maybe this isn't the right place to discuss this but why isn't this function in helperFunctions/config.py or why isn't `load_config()` from that module used? Seems a bit redundant.

Reply:
Keep in mind that we are still setting up python dependencies at this point

Reply:
Indeed, the config helper depends on the process helper which itself depends on _psutil_. _psutil_ though is not present at link time of the installation on a clean system.

Reply:
that is unfortunate indeed